orchestra/static/orchestra/common/js/orchestra.services.js

```diff
@@ -67,3 +67,33 @@ serviceModule.factory('orchestraService', function() {
   };
   return orchestraService;
 });
+
+serviceModule.factory('orchestraTasks', function($http) {
+
+  var newTask = function(taskType) {
```

Comment:
At some point we should probably have an API service so we have all our URLs in one place, but totally good for now!